Update description of node selector methods (#2974)
dbt-labs/docs.getdbt.com: website/docs/reference/node-selection/methods.md

### The "path" method
The `path` method is used to select models/sources defined at or under a specific path.

> For models, the definition lies in SQL/Python files, not the configuration YAML.

Reviewer: ```suggestion
Model definitions are in SQL/Python files (not YAML), and source definitions are in YAML files.
```

> For sources, the definition lies in YAML files.

Reviewer: ```suggestion

```
